dotCMS/src/main/java/com/dotmarketing/util/MaintenanceUtil.java

```diff
@@ -689,9 +689,14 @@ public static void fixImagesTable() throws SQLException{
 	 *             An error occurred when executing the fix queries.
 	 */
 	public static void deleteOrphanContentTypeFields() throws SQLException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.sql.SQLException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)